### PR TITLE
Jaccard Distance Utility

### PR DESCRIPTION
**Pull request by Mayukh-G**
## Describe your changes
- Added Jaccard Distance utility and related methods.
- Added Jaccard Distance utility test
## Issue ticket number and link
- #84 
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: src/test/java/org/products/productreviews/unittests/app/util/JDistanceTest.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        A.addReview(rev2);
+
+        B.addReview(rev3);
+        B.addReview(rev4);
+
+        JDistance jDistanceAB = new JDistance(A, B);
+        float result = jDistanceAB.distance();
+        assertEquals(1f/3f, result);
+
+    }
```
Comment (Yasir-Sh) [inline src/test/java/org/products/productreviews/unittests/app/util/JDistanceTest.java:127]: Maybe a test with 3 accounts? Just to test if the "similar" account actually has a smaller distance.

Comment (Mayukh-G) [reply src/test/java/org/products/productreviews/unittests/app/util/JDistanceTest.java:127]: Added in [5484212](https://github.com/TrustyUnitReview/ProductReview/pull/87/commits/54842126deb8f3ad0b6c89496025c7f60460c466)

**Review by Yasir-Sh — DISMISSED** [review]
Looks good

**Review by Yasir-Sh — DISMISSED** [review]
Looks good

**Review by Yasir-Sh — APPROVED** [review]

**Review by Yasir-Sh — DISMISSED** [review]

**Review by JoshBraddon — APPROVED** [review]
LGTM!

**Comment by Mayukh-G** [comment]
Github Bug not allowing merge

**Pull request closed without merging**